go/cmd/dolt/commands/engine/sql_print.go

```diff
@@ -216,9 +241,9 @@ type nullWriter struct{}
 func (n nullWriter) WriteSqlRow(ctx *sql.Context, r sql.Row) error { return nil }
 func (n nullWriter) Close(ctx context.Context) error               { return nil }
 
-func printEmptySetResult(start time.Time) {
+func printEmptySetResult(start time.Time, warning string) {
 	seconds := secondsSince(start, time.Now())
-	cli.Printf("Empty set (%.2f sec)\n", seconds)
+	cli.Printf("Empty set%s (%.2f sec)\n", warning, seconds)
```

Comment:
Formatting mechanisms work better if you don't depend on spaces in the argument strings. So rather than prefix the warning string with a space, ensure it's trimmed, and use the format string: 
```suggestion
	cli.Printf("Empty set %s (%.2f sec)\n", warning, seconds)
```